src/OrchardCore/OrchardCore.Email.Core/Services/SmtpService.cs

```diff
@@ -210,7 +210,8 @@ private bool CertificateValidationCallback(object sender, X509Certificate certif
 
             return false;
         }
-        private async Task<string> SendOnlineMessage(MimeMessage message)
+
+        protected virtual async Task<string> SendOnlineMessage(MimeMessage message)
```

Comment:
```suggestion
        protected virtual async Task<string> SendOnlineMessageAsync(MimeMessage message)
```

Reply:
I thought you will suggest that dean ;) I already suggested in the description above